src/main/java/org/jacp/repositry/QuestionRepositoryCriteria.java

```diff
@@ -0,0 +1,42 @@
+package org.jacp.repositry;
+
+import jakarta.persistence.EntityManager;
+import jakarta.persistence.PersistenceContext;
+import jakarta.persistence.criteria.CriteriaBuilder;
+import jakarta.persistence.criteria.CriteriaQuery;
+import jakarta.persistence.criteria.Predicate;
+import jakarta.persistence.criteria.Root;
+import org.jacp.entity.CompetitionEntity;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+/**
+ * @author saffchen created on 11.03.2024
+ */
+@Repository
+public class QuestionRepositoryCriteria {
```

Comment:
Minor: naming is little bit strange, it should be either Repository or Criteria.

Reply:
Fixed